Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -414,7 +414,7 @@ inline HRESULT PopEndDraw() {
     template <typename Lambda> // Lambda takes the form HRESULT (*)(CGContextRef, ID2D1DeviceContext*)
     HRESULT Draw(_CGCoordinateMode coordinateMode, CGAffineTransform* additionalTransform, Lambda&& drawLambda);
 
-    HRESULT DrawGeometry(_CGCoordinateMode coordinateMode, ID2D1Geometry* pGeometry, CGPathDrawingMode drawMode);
+    HRESULT DrawGeometry(_CGCoordinateMode coordinateMode, ID2D1Geometry* pGeometry, CGPathDrawingMode drawMode, bool isPathGeometry);
```

Comment:
But the fill mode for a simple shape like a rectangle doesn't matter, as it has no overlapping sections.